.github/workflows/check-unity-sdk-for-updates.yml

```
@@ -37,7 +41,13 @@ jobs:
             exit 1
           fi
           echo "LATEST_ANDROID_SDK_VERSION=$LATEST_VERSION" >> "$GITHUB_ENV"
-          echo "HAS_CHANGES=$(git status --porcelain -z | tr '\0' ' ')" >> "$GITHUB_ENV"
+          
+          CURRENT_VERSION=$(grep -oP "implementation 'com.adgem.android:adgem-sdk:\K[^']+" AndroidManifest.xml)
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)